snooty.toml

```diff
@@ -40,6 +40,7 @@ not-available = "N/A"
 package = "MongoDB.EntityFrameworkCore"
 api-root = "https://mongodb.github.io/mongo-efcore-provider/{+full-version+}/api"
 driver-api-root = "https://mongodb.github.io/mongo-csharp-driver/3.0.0/api"
+driver-docs-root = "https://www.mongodb.com/docs/drivers/csharp/current"
```

Comment:
S: are there other places that link between these docs sets? Maybe you can create a ticket in the backlog to update any other instances that use this link

Reply:
There was only one other instance so just updated it here